base_classes/NXbeam.nxdl.xml

```diff
@@ -179,13 +217,49 @@
             <dim index="1" value="nP"/>
             <dim index="2" value="2"/>
         </dimensions>
+        <attribute name="units" type="NX_CHAR">
+            <doc>
+                 The units for this observable are not included in the NIAC list.
+                 Responsibility on correct formatting and parsing is handed to the user
+                 by using `NX_ANY`. Correct parsing can still be implemented by using
+                 this attribute.
+                 
+                 | Fill with:
+                 
+                 * The unit unidata symbol if the unit has one (Example: T for the unit of magnetic flux density tesla).
+                 * The unit unidata name if the unit has a name (Example: farad for capacitance).
+                 * A string describing the units according to unidata unit operation notation, if the unit is a complex combination of named units and
+                   does not have a name.
+                 
+                 Example: for lightsource brilliance (SI) 1/(s.mm2.mrad2).
+                 Here: SI units are V2/m2.
+            </doc>
+        </attribute>
```

Comment:
Delete this attribute block, and change units for `incident_polarization` to `V^2/m^2`.